src/atomate2/vasp/flows/magnetism.py

```diff
@@ -0,0 +1,109 @@
+"""VASP-specific flows for magnetism-related calculations."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Literal, Sequence
+
+from atomate2.common.flows.magnetism import (
+    MagneticOrderingsMaker as MagneticOrderingsMakerBase,
```

Comment:
DFT-independent `BaseMaker` are put in `common` and callable under `vasp/flows`. A good example to follow for future workflow writing.

Reply:
I think we're all good here!